site/docs/components/input/examples.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Input
+layout: DetailComponent
+sidebar:
+  label: Examples
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="input" exampleName="Primary" >
+
+### Primary
+
+Input comprises value text and a focus ring. The component is initially empty and then, once it has focus, it accepts text on a single line only.
+
+The default variant for Input is "primary".
+
+- Input is responsive to the size of its parent container, filling it 100%. To disable responsiveness, apply a fixed width and height using the CSS API.
+
+</LivePreview>
+<LivePreview componentName="input" exampleName="Secondary" >
+
+### Secondary
+
+Input has a "secondary" variant.
+
+For recommendations on variant choice, see the [Forms pattern](salt/patterns/forms).
+
+</LivePreview>
+<LivePreview componentName="input" exampleName="Disabled" >
+
+### Disabled
+
+You can disable Input, with no resultant action when the user interacts with it.
+
+</LivePreview>
+<LivePreview componentName="input" exampleName="Readonly" >
+
+### Readonly
+
+You can set Input to a read-only state. The value text can’t be edited in this state, but it can be highlighted and copied.
+
+By default, the marker is set to an em dash. If your application requires a different marker it can be set via the emptyReadOnlyMarker prop.
+
+- Use the read-only state when the value is necessary for the users flow or current task but cannot be edited, for example, because of user permissions.
+
+</LivePreview>
+<LivePreview componentName="input" exampleName="EmptyReadonlyMarker" >
+
+### Placeholder
+
+You can use a placeholder to prompt user input if no default value is provided.
+
+- Placeholder Text should never be used to provide the user with contextual help since it will be removed when the user starts typing and does not meet minimum contrast requirements. Doing so is a WCAG failure. Instead, use Helper Text beneath the field to provide instructions or directions which are necessary to complete the field, and reserve the Placeholder Text to support the help message, or provide an example of the suggested content.
```

Comment:
```suggestion
- Placeholder text should never be used to provide the user with contextual help since it will be removed when the user starts typing and does not meet minimum contrast requirements. Doing so is a WCAG failure. Instead, use helper text beneath the field to provide instructions or directions which are necessary to complete the field, and reserve the placeholder text to support the help message, or provide an example of the suggested content.
```